backend/src/email/utils/callback/parsers/ses.ts

```diff
@@ -183,10 +181,37 @@ const parseEvent = async (
 }
 
 const parseRecord = async (record: SesRecord): Promise<void> => {
-  if (!(record.SignatureVersion === '1' && (await validateSignature(record)))) {
-    throw new Error(`Invalid record`)
-  }
   const message = JSON.parse(record.Message)
+  const {
```

Comment:
Minor nit; this validation chunk here is quite lengthy. For readability, might want to extract this into a separate function. Something also to consider is whether we can introduce stronger types here too.